.github/workflows/run-tests.yml

```diff
@@ -111,11 +111,12 @@ jobs:
           path: ~/downloads/tarball
       - name: Echo tarball download location
         run: echo ${{ steps.download.outputs.download-path }}
+        # TODO: update ASAP after April Release
```

Comment:
What's the update that needs to happen here?

Reply:
oh good call - this isn't necessary with the latest change. I'll remove!